quic/s2n-quic-tls/src/session.rs

```diff
@@ -88,6 +88,14 @@ impl tls::TlsSession for Session {
             .tls_exporter(label, context, output)
             .map_err(|_| tls::TlsExportError::failure())
     }
+
+    fn cipher_suite(&self) -> CipherSuite {
+        if let Ok(cipher_suite) = self.connection.cipher_suite() {
```

Comment:
seems better to store the expected cipher suite on the session state, rather than parsing it from the connection?

Reply:
just to make sure I'm understanding, that would be done inside of `on_secret` in `callback.rs` by changing `fn get_algo_type` to also return the cipher suite?

Reply:
yeah that's what i was thinking